ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_service_insertion.schema.yml

```
@@ -8,7 +8,44 @@ type: dict
 keys:
   router_service_insertion:
     type: dict
-    description: Configure network services inserted to data forwarding
+    description: Configure network services inserted to data forwarding.
     keys:
       enabled:
         type: bool
+      connections:
+        type: list
+        primary_key: name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Connection name.
+            ethernet_interface:
+              type: dict
+              description: |-
+                Outgoing physical interface or subinterface to use for the connection.
+                If both ethernet interface and tunnel interface are configured, ethernet interface will be used.
+              keys:
+                name:
+                  type: str
+                  description: e.g. Ethernet2/2.2
```

That's not what I suggested I think
```suggestion
                  description: e.g. Ethernet 2 or Ethernet2/2.2
```

---

Corrected,